src/TLuaInterpreter.cpp

```diff
@@ -4911,7 +4913,7 @@ int TLuaInterpreter::getRooms( lua_State *L )
 // returned.
 int TLuaInterpreter::getAreaExits( lua_State *L )
 {
-    int area;
+    int area = 0;
```

Comment:
I added the `= 0;` before I added the `return;`. Coverity would probably get either one right and not warn.